tailcfg/tailcfg.go

```diff
@@ -80,7 +80,8 @@ type CapabilityVersion int
 //   - 41: 2022-08-30: uses 100.100.100.100 for route-less ExtraRecords if global nameservers is set
 //   - 42: 2022-09-06: NextDNS DoH support; see https://github.com/tailscale/tailscale/pull/5556
 //   - 43: 2022-09-21: clients can return usernames for SSH
-const CurrentCapabilityVersion CapabilityVersion = 43
+//   - 44: 2022-09-21: CapabilityDataPlaneAuditLogs, added Node.DataPlaneAuditLogID
+const CurrentCapabilityVersion CapabilityVersion = 44
```

Comment:
I don't think you need to bump this. The client can't do anything new as of this commit that the server would be interested in.

Reply:
OK.